src/index.tsx

```diff
@@ -1,13 +1,25 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { Container } from '@mui/material';
```

Comment:
we won't have app file as it represent that all pages start from there, but since we have routing it means each pages starts from different component (not the app component)